.github/workflows/verifyimage.yml

```diff
@@ -66,7 +66,7 @@ jobs:
 
       - name: Run ${{ matrix.image }}${{ matrix.variant }}
         run: |
-          docker run -d --name ${{ matrix.image }}${{ matrix.variant }}-test "${{ env.REPO }}:${{ matrix.image }}${{ matrix.variant }}"
+          docker run --pull "never" -d --name ${{ matrix.image }}${{ matrix.variant }}-test "${{ env.REPO }}:${{ matrix.image }}${{ matrix.variant }}"
```

Comment:
Will this not make the job fail? Why is this an issue anyway?

Reply:
Excellent questions. 

No, as you can see from the latest run, it works perfectly. It means "do not go to dockerhub (or other registries) to get images". All images should be loaded locally by now.

I found this previous run: https://github.com/coreruleset/modsecurity-docker/actions/runs/3063111140/jobs/4944831699, where the tags were created wrongfully. The next step was using the correct tags, but because the previous step didn't generated the correct tags, the image was pulled from upstream instead of using the local one. And the pipeline succeeded, but was generating the wrong tags. With this I'm trying to prevent that behavior.

Reply:
Also added an extended tag so it will be using the tags with the date of the build included.

Reply:
> No, as you can see from the latest run, it works perfectly. It means "do not go to dockerhub (or other registries) to get images". All images should be loaded locally by now.

I see. So it's a failsafe.

> Also added an extended tag so it will be using the tags with the date of the build included.

I tried to understand what you're doing there but I'm not really sure. AFAICT, the resulting `docker run` command is identical. At least, before, you'd see the tag name as part of the `docker run` command, now, the log only shows `"$TAG"`.

Reply:
Yeah, we can echo that:
``
❯ docker image inspect -f '{{ json .RepoTags }}' owasp/modsecurity:nginx | jq -r '.[0]'
owasp/modsecurity:3-202209140709
```

Reply:
Ah, yes. Sounds good.

Reply:
Echoed some informational comment about the tag used.